plugin/suda.vim

```diff
@@ -6,3 +6,21 @@ let g:loaded_suda = 1
 if get(g:, 'suda_startup', 1)
   call suda#init()
 endif
+
+if exists('g:suda_hijack_write')
+  cabbrev w W
+  cabbrev wa Wa
+  cabbrev wq Wq
+  cabbrev wqa Wqa
+  command! W execute suda#write_wrapper('')
+  command! Wa execute suda#write_wrapper('a')
+  command! Wq execute suda#write_wrapper('q')
+  command! Waq execute suda#write_wrapper('aq')
```

Comment:
The idea is, that with `g:suda_hijack_write` set, when a user types `:w` (lowercase) it will be replaced by the corresponding abbreviation to the command and he won't even feel that he's using the almighty suda plugin.

Reply:
> he won't even feel that he's using the almighty suda plugin.

Even user won't use those commands, those commands will be popped up on the command completions and `W`, `Wa`, ...etc don't tell which plugin uses that command. If the command names are `SudaWrite` or whatever, users will notice that the commands are defined from suda.vim

Reply:
Fine with me :cat2: